containers/jekyll/.devcontainer/base.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:2
+COPY library-scripts/meta.env /usr/local/etc/vscode-dev-containers
+
+# ENV Variables required by Jekyll
+ENV LANG=en_US.UTF-8 \
+    LANGUAGE=en_US:en \
+    TZ=Etc/UTC \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US
+
+# Install bundler, latest jekyll, and github-pages for older jekyll
+RUN gem install bundler jekyll github-pages
```

Comment:
This appears to work, installing both Jekyll 3 and Jekyll 4 as desired to speed up the `bundle install` process.